x-pack/plugins/endpoint/server/test_data/all_metadata_data.json

```diff
@@ -1,217 +1,205 @@
 {
-  "took" : 343,
-  "timed_out" : false,
-  "_shards" : {
-    "total" : 1,
-    "successful" : 1,
-    "skipped" : 0,
-    "failed" : 0
+  "took": 343,
```

Comment:
can you explain the change here? also, anyone know why this is large and not gzipped?

Reply:
@jonathan-buttner This will be on me to fix.

Reply:
@oatkiller most of the changes are just spacing and renaming the index being used:

![image](https://user-images.githubusercontent.com/56361221/79476865-85efc180-7fd7-11ea-8504-580959b480b6.png)

my vs code default formatter is removing the spaces like these, I can revert them if you prefer:

![image](https://user-images.githubusercontent.com/56361221/79476792-6eb0d400-7fd7-11ea-8f33-ce3fc208b9d4.png)

I believe this is not gzipped because it's directly imported in typescript code here:
https://github.com/elastic/kibana/pull/63016/files/2829689ae9674970d57644212863734d34d2cd5b#diff-b1e4abedd6d2153cf169796ebae56a93R27
![image](https://user-images.githubusercontent.com/56361221/79477069-ba637d80-7fd7-11ea-8f13-1c53f3024f1b.png)

@nnamdifrankie yeah if you could fix it that'd be great!